src/screens/profile/index.js

```diff
@@ -0,0 +1,342 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/styles";
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardActions,
+  Divider,
+  Grid,
+  Button,
+  TextField,
+  Avatar,
+  Typography,
+  LinearProgress,
+} from "@material-ui/core";
+import Topbar from "../../components/Topbar/index";
+import Link from "@material-ui/core/Link";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 10,
+    margin: 10,
+  },
+  details: {
+    display: "flex",
+  },
+  avatar: {
+    marginLeft: "auto",
+    height: 110,
+    width: 100,
+    flexShrink: 0,
+    flexGrow: 0,
+  },
+  progress: {
+    marginTop: "2px",
+  },
+  uploadButton: {
+    marginRight: "2px",
+  },
+  profile2: {
+    height: 1100,
+    width: 800,
+  },
+}));
+
+var count = 0;
+
+const Profile = (props) => {
+  const { className, ...rest } = props;
+  const dpInput = React.createRef();
+
+  const classes = useStyles();
+
+  const [values, setValues] = useState({
+    name: props.name,
+    userName: props.userName,
+    email: props.mail,
+    fblink: props.fbLink,
+    followers: props.followers,
+    following: props.following,
+    nickname: props.nickname,
+    hall: props.hall,
+  });
+  document.body.style = "background:#f5f5ef ; margin-bottom: 60px;";
+
+  //Handle change in text fields and update profile completeness
+  const handleChange = (event) => {
+    setValues({
+      ...values,
+      [event.target.name]: event.target.value,
+    });
+    count = 0;
+    for (var key in values) {
+      if (values[key]) {
+        count++;
+      }
+    }
```

Comment:
A cleaner way for the profile completeness is to keep `count` as a state variable in the component (and rename it to `completenessIndex` or something like that) and here, in this function, set this state variable to the value of `count`.